app/views/admin/editorial_remarks/new.html.erb

```diff
@@ -1,21 +1,34 @@
-<% page_title "New Editorial Remark" %>
+<% content_for :page_title, "New editorial remark" %>
```

Comment:
Ditto for the wording used here.

Also, we should probably set the page 'context' to the edition title, for consistency with other Design System'd pages – for example, the 'Topic taxonomy tags' page

https://github.com/alphagov/whitehall/blob/2b90ee26775ea4484312ce9d6c9201a8a509d9c3/app/views/admin/edition_tags/edit.html.erb#L1-L3

Again – this might be something you'd prefer to cover in the [Move Notes/History/Fact-checking tabs](https://trello.com/c/xiy8xs27/888-move-notes-history-fact-checking-tabs-to-the-design-system) card rather than this PR.